File: 프로그래머스/lv0/120826. 특정 문자 제거하기/특정 문자 제거하기.js
function solution(my_string, letter) {
    my_string = my_string.split("");
    
    for (let i = 0; i < my_string.length; i++) {
        if (my_string[i] == letter) {
            my_string[i] = "";
        }        
    }  
    return my_string.join("");
}

File: 프로그래머스/lv0/120826. 특정 문자 제거하기/특정 문자 제거하기.java
class Solution {
    public String solution(String my_string, String letter) {
        return my_string.replaceAll(letter, "");
    }
}

/*
class Solution {
    public String solution(String my_string, String letter) {
        String[] answer = new String[my_string.length()];
        answer = my_string.split("");

        for (int i = 0; i < answer.length; i++) {
            if (answer[i].equals(letter)) {
                answer[i] = "";
            }
        }

        return String.join("", answer);
    }
}
*/